plugins/usd/configs/config.d/10_usd.json

```diff
@@ -0,0 +1,6 @@
+{
+  ".*(usd)[acz]?":
+  {
+    "load-plugins": "usd"
```

Comment:
No. The up direction for example is specified within the file...  
I think it would make sense for full scenes to configure the up direction in the importer at runtime.  
But I won't do that in this PR.